documentation/source/tutorials/pages/registering-different-data-from-the-same-session.rst

```diff
@@ -0,0 +1,189 @@
+                           -owarp warp_template2mt.nii.gz -qc ~/qc_singleSubj
+
+   # Input arguments:
+   #   - i: Source image. Here, it is the PAM50 template taken from the SCT installation directory. The T2 version
+   #        of the template is used due to its similarity in contrast to the MT1 data.
+   #   - iseg: Segmentation corresponding to the source image. Here, it is the segmented spinal cord volume from
+   #           the PAM50 template, taken from the SCT installation directory.
+   #   - d: Destination image.
+   #   - dseg: Segmentation corresponding to the destination image.
+   #   - m: Mask image, which is used on the destination image to improve the accuracy over the region of interest.
+   #   - initwarp: Initial warping field to apply to the source image. Here, we supply the 'warp_template2anat.nii.gz'
+   #               file that was generated in the previous tutorial. Because we begin with the 'Template->T2'
+   #               transform already applied, the warping field that is generated here will be 'Template->T2->MT1'
+   #               a.k.a. 'Template->MT1'.
+   #   - param: Here, we will tweak the default registration parameters to specify a different nonrigid deformation.
+   #            The important change is 'algo=centermass': Because the template object is already "preregistered"
+   #            from the previous tutorial (see '-initwarp'), the benefits of the default 'algo=centermassrot' have
+   #            already been applied. So, we specify 'algo=centermass' in step 1 to exclude the unnecessary rotation.
+   #   - owarp: The name of the output warping field. This is optional. If not supplied, the filename would be
+   #            generated from the filenames '-i' and '-d', which in this case would be 'warp_PAM50_t22mt1.nii.gz'.
+   #   - qc: Directory for Quality Control reporting. QC reports allow us to evaluate the results slice-by-slice.
+
+   # Output files/folders:
+   #   - mt1_reg.nii.gz: TODO: Empty file. How to explain?
+   #   - PAM50_t2_reg.nii.gz: The PAM50 template image, registered to the space of the MT1 image.
+   #   - warp_template2mt.nii.gz: The warping field to transform the PAM50 template to the MT1 space.
+
+.. _mt-registraton-without-anat:
+
+Method 2: Registering MT data without anatomical images
+=======================================================
+
+In the case that you have only the MT data without the anatomical data, you can still perform registration. To do so, all you will need to do is apply the same vertebral labeling and template registration steps that were covered in :ref:`registration-to-template`.
+
+First, we create one or two labels in the metric space. For example, if you know that your FOV is centered at C3/C4 disc, then you can create a label automatically with:
+
+.. code:: sh
+
+   sct_label_utils -i mt1_seg.nii.gz -create-seg -1,4 -o label_c3c4.nii.gz
```

Comment:
This command didn't work, here is my terminal output:

```
(venv_sct) sabeda@DESKTOP-JQ8A4UV:/mnt/c/Users/sb199/Projet3_data/sct_course_london20/single_subject/data/mt$ sct_label_utils -i mt1_seg.nii.gz -create-seg -1,4 -o label_c3c4.nii.gz

--
Spinal Cord Toolbox (git-master-a62fb672c602b9863a8c03521eb272bfd7bad9e1*)

sct_label_utils -i mt1_seg.nii.gz -create-seg -1,4 -o label_c3c4.nii.gz
--

Traceback (most recent call last):
  File "/mnt/c/Users/sb199/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_label_utils.py", line 413, in <module>
    main(sys.argv[1:])
  File "/mnt/c/Users/sb199/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_label_utils.py", line 242, in main
    raise DeprecationWarning("The use of '-1' for '-create-seg' has been deprecated. Please use "
DeprecationWarning: The use of '-1' for '-create-seg' has been deprecated. Please use '-create-seg-mid' instead.
```

Reply:
Aha! Right. I remember this change. I'm glad I added a warning for this. :sweat_smile: 

Thanks for letting me know. :smile:

Reply:
You're welcome! You can hit me up if you have other things to test out 🙂